.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "develop" ]
+  pull_request:
+    branches: [ "develop" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
```

Comment:
The rust toolchain has to be installed.

Reply:
Interesting! My assumption was since this is github hosted and this was basically their default template that was handled. Is that incorrect?

Reply:
This could be a good option https://github.com/actions-rust-lang/setup-rust-toolchain